src/indigo/components/NewLineToBr.js

```diff
@@ -4,23 +4,21 @@ import PropTypes from 'prop-types';
 class NewLineToBr extends React.Component {
   render() {
     const lines = this.props.text.split('\n');
-    let result = [];
+    const result = [];
```

Comment:
toto ale musi byt `let`, nie?

Reply:
Vidím tam pak jenom push. Tak by to mělo být fajn. Nebo něco nevidím?

Reply:
jo, pozeram, ze to funguje ... ale mal by tam byt imho `let` ... teraz mi to evokuje, ze sa do `result` uz nizsie v kode nebude zapisovat ... `result['hovno'] = 'hovno';` ale prejde a nepinda ... zasrany javascript :roll_eyes:

Reply:
Na toto bude v eslint určitě pravidlo. Zda to kontrolovat a přepisovat nebo ne.